### ISSUE TITLE
Q2-Beta7: Electron build fails

### ISSUE THREAD

**Issue by McShelby** [quasarframework/quasar#8521]
**Describe the bug**
Creating a plain demo project, adding electron and invoking a build fails to create the app executable. There are artifacts in `dist/electron/UnPackaged` but no executable is created. Using `quasar dev -m electron` works fine.

**To Reproduce**
```bash
mkdir q2b7
cd q2b7
quasar create --branch next # using npm option
quasar mode add electron
quasar build -m electron
```

**Screenshots**
````bash
 Build mode........ electron
 Pkg quasar........ v2.0.0-beta.7
 Pkg @quasar/app... v3.0.0-beta.7
 Debugging......... no
 Publishing........ no

 Configured browser support (at least 87.91% of global marketshare):
 · Chrome for Android >= 88
 · Firefox for Android >= 85
 · Android >= 81
 · Chrome >= 78
 · Edge >= 85
 · Firefox >= 76
 · iOS >= 10.3
 · Opera >= 69
 · Safari >= 11

 App · Reading quasar.conf.js
 App · Transpiling JS (Babel active)
 App · Installing required Electron bundler (electron-packager)...
 App · [sync] Running "npm install --save-dev electron-packager@^14.1.1"

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules\webpack-dev-server\node_modules\fsevents): 
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules\watchpack-chokidar2\node_modules\fsevents):npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

+ electron-packager@14.2.1
added 29 packages from 26 contributors and audited 1503 packages in 11.149s

106 packages are looking for funding
  run `npm fund` for details

found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
 App · Chaining Renderer process Webpack config
 App · Chaining Preload process Webpack config
 App · Chaining Main process Webpack config
 App · Cleaned build artifact: "E:\ProgalSrc\IM_Web\q2b7\dist\electron"
 App · Generating Webpack entry point
 App · Compiling with Webpack...

 • Compiling:
 ├── Renderer process ████████             35% building 214/215 modules 1 active
 ├── Preload process  ████████████████████ 95% emitting
 └── Main process     ████████████████████ 100% done in 6894 ms


 • Compiling:
 ├── Renderer process ███████████████████  92% additional asset processing copy-webpack-plugin
 ├── Preload process  ████████████████████ 100% done in 7923 ms
 └── Main process     ████████████████████ 100% done in 6894 ms


 • Compiling:
 ├── Renderer process ████████████████████ 100% done in 11811 ms
 ├── Preload process  ████████████████████ 100% done in 7923 ms
 └── Main process     ████████████████████ 100% done in 6894 ms

 App · Added build artifact "E:\ProgalSrc\IM_Web\q2b7\dist\electron"

 Renderer process summary for css/js/json (the rest are omitted):
 ╔════════════╤═══════════╤══════════╗
 ║      Asset │      Size │  Gzipped ║
 ╟────────────┼───────────┼──────────╢
 ║    app.css │   0.00 KB │  0.02 KB ║
 ║ vendor.css │ 187.60 KB │ 33.29 KB ║
 ╟────────────┼───────────┼──────────╢
 ║       2.js │   0.56 KB │  0.38 KB ║
 ║       3.js │   3.84 KB │  1.37 KB ║
 ║       4.js │   0.74 KB │  0.46 KB ║
 ║     app.js │   3.21 KB │  1.55 KB ║
 ║  vendor.js │ 179.77 KB │ 63.44 KB ║
 ╚════════════╧═══════════╧══════════╝

 Preload process summary for css/js/json (the rest are omitted):
 ╔═════════════════════╤═════════╤═════════╗
 ║               Asset │    Size │ Gzipped ║
 ╟─────────────────────┼─────────┼─────────╢
 ║ electron-preload.js │ 0.96 KB │ 0.46 KB ║
 ╚═════════════════════╧═════════╧═════════╝

 Main process summary for css/js/json (the rest are omitted):
 ╔══════════════════╤═════════╤═════════╗
 ║            Asset │    Size │ Gzipped ║
 ╟──────────────────┼─────────┼─────────╢
 ║ electron-main.js │ 1.76 KB │ 0.86 KB ║
 ╟──────────────────┼─────────┼─────────╢
 ║     package.json │ 0.28 KB │ 0.21 KB ║
 ╚══════════════════╧═════════╧═════════╝

 App · Running "npm install --production"


> core-js@3.9.0 postinstall E:\ProgalSrc\IM_Web\q2b7\dist\electron\UnPackaged\node_modules\core-js
> node -e "try{require('./postinstall')}catch(e){}"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> https://opencollective.com/core-js 
> https://www.patreon.com/zloirock 

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)

npm notice created a lockfile as package-lock.json. You should commit this file.
added 2 packages from 1 contributor and audited 2 packages in 1.319s

2 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities


 App · Bundling app with electron-packager...

(node:56464) UnhandledPromiseRejectionWarning:   TypeError: bundler is not a function

  - index.js:161 
    [q2b7]/[@quasar]/app/lib/electron/index.js:161:13

  - new Promise

  - index.js:156 
    [q2b7]/[@quasar]/app/lib/electron/index.js:156:14

  - task_queues.js:93 processTicksAndRejections
    internal/process/task_queues.js:93:5


(Use `node --trace-warnings ...` to show where the warning was created)
(node:56464) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:56464) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
````

**Platform (please complete the following information):**
Quasar Version: 2.0.0-beta.7
@quasar/app Version: 3.0.0-beta.7
Quasar mode:
  [ ] SPA
  [ ] SSR
  [ ] PWA
  [x] Electron
  [ ] Cordova
  [ ] Capacitor
  [ ] BEX
Tested on:
  [ ] SPA
  [ ] SSR
  [ ] PWA
  [x] Electron
  [ ] Cordova
  [ ] Capacitor
  [ ] BEX
OS: Microsoft Windows [Version 10.0.19042.685]
Node: 14.15.5
NPM: 6.14.11
Electron: 11.3.0

**Comment by rstoenescu** [comment]
Just run the build command again until things are figured out.

**Comment by rstoenescu** [comment]
Fix will be available in q/app 3.0.0-beta.8
